tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,122 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+class BasePlasmaTest(object):
+    #Class defining all common tests for different setups of Plasma
+    #This can then be inherited for different Plasma setup
+
+    @pytest.fixture(scope="class", autouse=True)
```

Comment:
I think, we don't need to use `autouse=True` here as you are using the fixture directly in the tests.